### ISSUE TITLE
Buttons with 'Text scrolling effect' are (sometimes) broke with HA 2024.09

### ISSUE THREAD

**Issue by IzeQube** [Clooos/Bubble-Card#769]
**Describe the bug**  
If *Text scrolling effect* is enabled for the button it sometimes starts to scroll even if it isn't needed. F5 helps in that case.

I'm using Button Card in a 2 Column Section Dashboard.

The Problem started with HA 2024.09

**To Reproduce**  
Create a Button with enabled *Text scrolling effect* and reload the dashboard till it starts to scroll.

**Expected behavior**  

**Screenshot**
![image](https://github.com/user-attachments/assets/d6558391-091d-4c49-8246-b54c1a3a437e)

**Informations (please complete the following information):**  
 - OS: Windows, Android, iOS
 - Browser/App: Edge, Chrome, Safari
 - Bubble Card version: 2.2.4
 - Home Assistant version: 2024.09

**Comment by drupry** [comment]
Yes, I have same problem after update 2024.9 but only in mobile app in both Android and iOS, in web browser it's ok.

**Comment by realroywalker** [comment]
I'm seeing the same issue, not sure if it helps to debug the issue but for me it only appeared when I enabled the conditional visibility for a section on HA.
I have 3 sections, the first being a vertical menu, and the other 2 housing bubble buttons - as soon as I enable the conditional visibility for the first section so that it doesn't appear on mobile it causes buttons in the remaining 2 sections to scroll their text (even when not needed).
I guess that whatever method is used to hide the first sections screws up the width calculation for bubble and it thinks the screen is not wide enough?

**Comment by redfusion83** [comment]
Same problem here while using the companion app on Android. Note that it only occurs on Bubblecards that have sub buttons.

![Screenshot_20240910_063624_Home Assistant](https://github.com/user-attachments/assets/7cc956fb-4d64-482f-bd4d-45d7fd8de88e)

**Comment by Clooos** [comment]
Hi! Is that issue still happening? I was unable to reproduce it, it's possible that I've missed an HA update.

🤞

**Comment by drupry** [comment]
Not really, its happening when the page is not fully loaded, but after I think 2024.9.2 I don't have this problem anymore.

**Comment by Clooos** [comment]
I guess that I can close this issue then 🍻
